changelog.dd

```diff
@@ -23,6 +23,7 @@ $(BUGSTITLE Language Changes,
 $(BUGSTITLE Compiler Changes,
     $(LI $(RELATIVE_LINK2 deferred_alias, Analysis for aliases in imported modules is deferred.))
     $(LI $(RELATIVE_LINK2 native_tls_osx, Native TLS on OS X 64 bit.))
+    $(LI $(RELATIVE_LINK2 __FILE_FULL_PATH__, Special trait replaced by the source file's absolute file name.)
```

Comment:
A manifest constant? I'm not familiar with the term, what's the definition?  I got "special trait" from https://dlang.org/spec/traits.html , but it looks like that page say's it's actually a "special keyword".